source/includes/fundamentals/code-snippets/builders/AggBuilders.java

```diff
@@ -302,4 +302,13 @@ private void matchStage() {
         match(eq("title", "The Shawshank Redemption"));
         // end match
     }
+
+    private void documents() {
+        // begin documents
+        documents(Arrays.asList(
+                new Document("title", "The Shawshank Redemption"),
+                new Document("title", "Back to the Future"),
+                new Document("title", "Jurassic Park")));
+        // end documents
```

Comment:
Q: Did you confirm that this is the correct syntax to create this stage?

Reply:
Just making sure -- it seems consistent with what is on this page, but doesn't exactly match the [tests from the feature PR](https://github.com/mongodb/mongo-java-driver/blob/a19fad2b3d93ee818132f14377e42cfaa268e8ee/driver-core/src/test/functional/com/mongodb/client/model/AggregatesTest.java#L170-L171).

Reply:
I confirmed that it _works_, but not whether it's the preferred syntax. I was planning to send to tech review after this.